### PR TITLE
Remove deprecated package symfony/templating

### PR DESCRIPTION
**Pull request by maxhelias**
- symfony/templating is no longer used for this bundle

### CODE REVIEW

File: Templating/Helper/BreadcrumbsHelper.php [hunk 1]
```diff
@@ -1,79 +1,78 @@
-<?php
-
-namespace WhiteOctober\BreadcrumbsBundle\Templating\Helper;
-
-use Symfony\Component\Templating\Helper\Helper;
-use Twig\Environment;
-use WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs;
-
-class BreadcrumbsHelper extends Helper
```
Comment (maxhelias) [inline Templating/Helper/BreadcrumbsHelper.php:9]: I don't know why there's all this diff, I just deleted the `extend Helper`

**Comment by mhujer** [comment]
@maxhelias Hello and thanks for the PR!

I've migrated the repo from Travis to Github workflows and did some of the changes that you proposed in separated commits:

* drop support for Symfony <4.4: c84865cd3735f1d8f2956454f848a161f1b548c2
* symfony/twig-bridge is required through symfony/twig-bundle: 4164b18473a9b30b4f1dc6e2a44e81b2345527ba
* building against PHP 8: 3ca36554a9a84fa4313ce6319f1161dcfab5471f

But:
* `twig/twig` needs to be specified because it constraints the compatibile versions `^1.38.1|^2.7.1|^3.0`
* I'm not sure about removing `symfony/templating` - is it related to this https://symfony.com/blog/new-in-symfony-4-3-deprecated-the-templating-component-integration? If so, can you please rebase the PR and clean it, so it contains only the changes related to removing the `symfony/templating`?

**Comment by maxhelias** [comment]
@mhujer I should have split into several commits but now it's ok.

For `twig/twig`, we can keep it but we should update the version to `^2.10|^3.0` to be in accordance with what symfony supports.

File: Templating/Helper/BreadcrumbsHelper.php (hunk 1 above)
Comment (mhujer) [reply Templating/Helper/BreadcrumbsHelper.php]: The file originally had CRLF newlines (Windows ones). I've resolved the problem by adding a separate commit just with the CRLF -> LF conversion

**Comment by mhujer** [comment]
@maxhelias thanks for the PR. Merged and released as 1.5.4.

I haven't changed the twig constraint as the `symfony/twig-bundle` 4.4 still allows Twig 1.x.

**Pull request merged**